.snyk

```diff
@@ -0,0 +1,17 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.13.5
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JAVA-COMBEUST-174815:
+    - '*':
+        reason: does not seem applicable
+        expires: 2019-07-28T20:07:33.140Z
```

Comment:
Why are you setting the expiry to one month from now? Is the idea to keep reevaluating these?

Reply:
ah, missed that. think its a default setting from the cli